CHANGELOG.md

```diff
@@ -1,3 +1,20 @@
+# Optimizely Csharp SDK Changelog
+
+## 3.2.0
+August 19th, 2019
+
+### New Features:
+- Introduced `UserEventFactory` which will return impression and conversion event.
```

Comment:
Should be in sync with the release notes from the JS SDK: https://github.com/optimizely/javascript-sdk/pull/354

Reply:
We could align on the style of these changelogs. JS's is verbose and reads like a summary/guide, but Java and this one are spartan.